src/main/java/com/um/inventoryservice/DataLayer/StockItem.java

```diff
@@ -19,5 +19,5 @@ public class StockItem {
     private int quantitySold;
     private double costPrice;
     private double sellingPrice;
-    private int quantity;
+    private int quantityInStock;
```

Comment:
Reason for attribute name change?

Reply:
was better naming, there is 2 variable processing quantities so now we know what both does.